tests/framework/installer/ceph_installer.go

```diff
@@ -620,40 +621,69 @@ func (h *CephInstaller) UninstallRookFromMultipleNS(systemNamespace string, name
 	}
 	checkError(h.T(), err, "cannot uninstall rook-operator")
 
-	h.k8shelper.Clientset.AdmissionregistrationV1beta1().ValidatingWebhookConfigurations().Delete("rook-ceph-webhook", nil)
-	h.k8shelper.Clientset.RbacV1().RoleBindings(systemNamespace).Delete("rook-ceph-system", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoleBindings().Delete("rook-ceph-global", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoleBindings().Delete("rook-ceph-mgr-cluster", nil)
-	h.k8shelper.Clientset.CoreV1().ServiceAccounts(systemNamespace).Delete("rook-ceph-system", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoles().Delete("rook-ceph-cluster-mgmt", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoles().Delete("rook-ceph-mgr-cluster", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoles().Delete("rook-ceph-mgr-system", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoles().Delete("rook-ceph-global", nil)
-	h.k8shelper.Clientset.RbacV1().Roles(systemNamespace).Delete("rook-ceph-system", nil)
-
-	h.k8shelper.Clientset.RbacV1().ClusterRoleBindings().Delete("rbd-csi-nodeplugin", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoles().Delete("rbd-csi-nodeplugin", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoleBindings().Delete("rbd-csi-provisioner-role", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoles().Delete("rbd-external-provisioner-runner", nil)
-
-	h.k8shelper.Clientset.RbacV1().ClusterRoleBindings().Delete("cephfs-csi-nodeplugin", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoles().Delete("cephfs-csi-nodeplugin", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoleBindings().Delete("cephfs-csi-provisioner-role", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoles().Delete("cephfs-external-provisioner-runner", nil)
-
-	h.k8shelper.Clientset.PolicyV1beta1().PodSecurityPolicies().Delete("rook-privileged", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoles().Delete("psp:rook", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoleBindings().Delete("rook-ceph-system-psp", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoleBindings().Delete("rook-csi-rbd-provisioner-sa-psp", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoleBindings().Delete("rook-csi-rbd-plugin-sa-psp", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoleBindings().Delete("rook-csi-cephfs-provisioner-sa-psp", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoleBindings().Delete("rook-csi-cephfs-plugin-sa-psp", nil)
-
-	h.k8shelper.Clientset.CoreV1().ConfigMaps(systemNamespace).Delete("csi-rbd-config", nil)
-	h.k8shelper.Clientset.CoreV1().ConfigMaps(systemNamespace).Delete("csi-cephfs-config", nil)
-
-	h.k8shelper.Clientset.RbacV1().ClusterRoleBindings().Delete("rook-ceph-object-bucket", nil)
-	h.k8shelper.Clientset.RbacV1().ClusterRoles().Delete("rook-ceph-object-bucket", nil)
+	err = h.k8shelper.Clientset.AdmissionregistrationV1beta1().ValidatingWebhookConfigurations().Delete("rook-ceph-webhook", nil)
```

Comment:
I believe the changes in this method are causing the integration test failures. If the resources are not found (perhaps the test didn't actually create it), we shouldn't fail the test. You might create a wrapper method that checks if an err is `errors.IsNotFound(err)` before asserting.

Reply:
used existing method to `checkError` for checking `errors.IsNotFound` before asserting.